r/R/dplyr.R

```diff
@@ -438,6 +440,48 @@ build_function_list <- function(FUN) {
   )
 }
 
+arrow_r_string_match_function <- function(FUN) {
+  function(pattern, x, ignore.case = FALSE, fixed = FALSE) {
+    if (ignore.case) {
```

Comment:
Seems like this logic can also be factored out instead of duplicated.

Reply:
Simplified and factored out in f05321292daa48bd0a58a151819e99ec6a14bc62